kodi_addon_checker/check_addon.py

```diff
@@ -347,10 +347,14 @@ def _check_complex_addon_entrypoint(report: Report, addon_path, parsed_xml, max_
             filepath = os.path.join(addon_path, library)
 
             if not os.path.isdir(filepath):
+                name, ext = os.path.splitext(filepath)
 
                 if os.path.exists(filepath):
```

Comment:
Please add an else condition where you basically say, that we can't find the file with that path and print the path.

Reply:
Whoops somehow missed that in the diff